### PR TITLE
Issue #129 : Added 'disclaimer' and 'terms' fields

### PR DESCRIPTION
**Pull request by fschiettecatte**
Added 'disclaimer' and 'terms' fields, see issue #129.

### CODE REVIEW

**Comment by Relequestual** [comment]
We should note that as per item 6 of the MME service agreement, this disclaimer should be shown with returned data.

**Comment by fschiettecatte** [comment]
@Relequestual - done.

File: search-api.md [hunk 1]
```diff
@@ -88,6 +90,15 @@ After receiving a request, the remote server can respond in one of two ways:
...
 ```
 
+#### Disclaimer
+* *Optional*
+* Disclaimer text for the service, this is optional and can be included in all requests and responses. If included it is assumed that it supersedes the disclaimer stored in the GitHub repository.
+* Note that the diclaimer should be shown with the returned data per item 6 of the MME service agreement.
+
+#### Terms
+* *Optional*
+* Terms and condittions text for the service, this is optional and can be included in all requests and responses.
```
Comment (sdumitriu) [inline search-api.md]: Typo in: condittions.

File: search-api.md [hunk 2]
```diff
@@ -88,6 +90,15 @@ After receiving a request, the remote server can respond in one of two ways:
 }
 ```
 
+#### Disclaimer
+* *Optional*
+* Disclaimer text for the service, this is optional and can be included in all requests and responses. If included it is assumed that it supersedes the disclaimer stored in the GitHub repository.
+* Note that the diclaimer should be shown with the returned data per item 6 of the MME service agreement.
```
Comment (sdumitriu) [inline search-api.md]: Maybe put a link to the agreement?

File: search-api.md [hunk 3]
```diff
@@ -88,6 +90,15 @@ After receiving a request, the remote server can respond in one of two ways:
 }
 ```
 
+#### Disclaimer
+* *Optional*
+* Disclaimer text for the service, this is optional and can be included in all requests and responses. If included it is assumed that it supersedes the disclaimer stored in the GitHub repository.
```
Comment (sdumitriu) [inline search-api.md]: If it can be included in responses, shouldn't it be documented in the response format as well?

**Comment by fschiettecatte** [comment]
All fixed, @sdumitriu you should feel free to pitch in and make such simple changes.

File: search-api.md (hunk 2 above)
Comment (Relequestual) [inline search-api.md]: I didn't think this was required as item 6 doesn't say anything extra, and an implementor should have already read and agreed to the service agreement.

Comment (fschiettecatte) [inline search-api.md]: GeneMatcher includes the disclaimer and guidelines for citation in the email we return to our submitters after they make an MME request. This information is currently stored in a table in our database but will likely be updated from the disclaimer included in the responses in the future. I know *we* have agreed to the service agreement but we need to make sure users get that too.

Comment (Relequestual) [inline search-api.md]: On DECIPHER, we display the disclaimers and the user must aggree / accept before they can see the data. I apprecaite this is not practical for GeneMatcher.

Comment (fschiettecatte) [inline search-api.md]: This is very good point, we have a EULA and guidelines in the FAQ they have to agree to when they sign up. I will check with Ada if we need to add disclaimers for each MME.

**Pull request merged**